releasenotes/notes/resonator-spectroscopy-89f790412838ba5b.yaml

```diff
@@ -0,0 +1,20 @@
+---
+fixes:
```

Comment:
This should not be fix, but upgrade (because Gaussian should work if background physics is Gaussian-ish). It would be great if you could write code block that shows how we reproduce old experiment by setting GaussianAnalysis class to experiment. You also need to mention upgrade of pulse duration unit for qubit and ef spectroscopy.

Reply:
I consider this as a fix since the background physics of a resonance should be a lorentzian which corresponds to the FFT of an exponential decay. I added an example for Gaussian here: 9990736